templates/core/terraform/network/network.tf

```diff
@@ -0,0 +1,53 @@
+resource "azurerm_virtual_network" "core" {
+  name                = "vnet-${var.resource_name_prefix}-${var.environment}-${var.tre_id}"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  address_space       = [var.address_space]
+}
+
+resource "azurerm_subnet" "bastion" {
+  name                 = "AzureBastionSubnet"
```

Comment:
Are we using Azure Bastion? I thought not.

Reply:
@marrobi requested we have subnet created for it; comments?

Reply:
It might be needed for administration - I usually se it deployed.

As it needs a dedicated subnet with specific name, my view is put the subnet in, as to add it post deployment is a headache to manage address space wise.

Reply:
The scenario here is for troubleshooting etc and not for the Virtual Desktop service.